pkg/commands/process/settings/rules/javascript/lang/weak_encryption.yml

```diff
@@ -0,0 +1,51 @@
+type: risk
```

Comment:
It's the default already, let's remove to not confuse people more than necessary